client/src/components/FeedbackPage/FeedbackTestimonialsSection/FeedbackTestimonialsSection.tsx

```diff
@@ -9,88 +9,39 @@ import RequestDataWrapper from 'components/RequestDataWrapper/RequestDataWrapper
 import Accordion from 'components/Accordion/Accordion';
 
 // Assets
-import { IconStar, IconBlobTwo } from 'assets/images/svg';
+import { IconStar } from 'assets/images/svg';
 
 // Utils
 import dateFormatter from 'utils/dateFormatter';
+import {
+  Testimonials,
+  testimonialsConfig,
+  generateTestimonialData
+} from './FeedbackTestimonialsSection.utils';
 
 // Styles
 import { Container, Flex } from 'styles/global';
 import * as S from '../FeedbackPage.styles';
 
-export enum Testimonial {
-  ratings = 'ratings',
-  suggestions = 'suggestions',
-  recommendations = 'recommendations',
-  more = 'more'
-}
-
-type Testimonials = Record<
-  Testimonial,
-  {
-    title: string;
-    list: {
-      name: string;
-      rating?: number;
-      message: string;
-      timestamp: Date;
-    }[];
-  }
->;
+const testimonialsConfigArray = Object.entries(testimonialsConfig).map(entry => ({
+  title: entry[1].title
+}));
 
 const FeedbackTestimonialsSection: FunctionComponent = () => {
   const theme = useAppSelector(selectTheme);
   const { data, loading, error } = useAppSelector(state => state.feedback);
   const [testimonials, setTestimonials] = useState<Testimonials>();
 
-  const testimonialsArray =
+  const testimonialsDataArray =
```

Comment:
please add types, we are still using typescript in project :)

Reply:
My bad again. It's just sometimes difficult to grasp where to place types...